examples/dos.jl

```diff
@@ -0,0 +1,32 @@
+# # Densities of states (DOS)
+# In this example, we'll plot the DOS, local DOS, and  projected DOS of Silicon.
+# DOS computation only supports finite temperature.
```

Comment:
You should run the SCF computation at zero temperature but the DOSes at finite temperature. Also be careful that the computation time is not too large (haven't checked)